pyqtgraph/functions.py

```diff
@@ -1129,7 +1129,7 @@ def makeARGB(data, lut=None, levels=None, scale=None, useRGBA=False, output=None
             raise Exception('levels argument is required for float input types')
     if not isinstance(levels, xp.ndarray):
         levels = xp.array(levels)
-    levels = levels.astype(xp.float)
+    levels = levels.astype(xp.float32)
```

Comment:
why 32, not 64?

Reply:
There was discussion about that here: https://github.com/pyqtgraph/pyqtgraph/pull/1518#issuecomment-769602093

Reply:
The discussion was concerning the choice of image data used for benchmarking. Users are still free to supply float64 images. So in this case, casting levels to float32 unconditionally might not be safe?

Reply:
Fix added to #1577